BrainCloudClient/src/BrainCloud/Client/BrainCloud/BrainCloudWrapper.cs

```diff
@@ -75,7 +75,7 @@ public class BrainCloudWrapper
     private static BrainCloudWrapper _instance = null;
     private static bool _applicationIsQuitting = false;
 
-    private BrainCloudClient _client = null;
+    public BrainCloudClient client = null;

```

Comment:
this would end up being Client... which we already have :) 

so feel free to keep this as a private reference, if you wish.. or more C# way 

extend the Public BrainCloudClient Client {get; private set;} <-- much cleaner, public getter, and a private setter